crates/cairo-lint-core/src/lints/eq_op.rs

```diff
@@ -0,0 +1,67 @@
+use cairo_lang_defs::plugin::PluginDiagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_syntax::node::TypedSyntaxNode;
+use cairo_lang_syntax::node::ast::{BinaryOperator, Expr, ExprBinary};
+use cairo_lang_syntax::node::db::SyntaxGroup;
+use cairo_lang_syntax::node::ids::SyntaxStablePtrId;
+
+const DIV_EQ_OP: &str = "Division with identical operands, this operation always results in one (except for zero) and \
+                         may indicate a logic error";
+const EQ_COMP_OP: &str =
+    "Comparison with identical operands, this operation always results in true and may indicate a logic error";
+const EQ_DIFF_OP: &str =
+    "Subtraction with identical operands, this operation always results in zero and may indicate a logic error";
+const EQ_BITWISE_OP: &str = "Bitwise operation with identical operands, this operation always results in the same \
+                             value and may indicate a logic error";
+const EQ_LOGICAL_OP: &str = "Logical operation with identical operands, this operation always results in the same \
+                             value and may indicate a logic error";
```

Comment:
make it just 1 message, for example:
`Operation with identical operands, this operation always results in one (except for zero) and may indicate a logic error`